tts/global.def.lua

```diff
@@ -42,3 +42,34 @@ function printToAll(message, color) end
 ---@return string
 function stringColorToRGB(player_color) end
 
+
+---@shape tts__SnapPoint
+---@field position tts__Vector Position of the snap point. The position is relative to the entity's center (a local position).
+---@field rotation tts__Vector Rotation of the snap point. The rotation is relative to the entity's rotation (a local rotation).
+---@field rotation_snap boolean If the snap point is a "rotation" snap point.
+
+---@overload fun(parameters: table):string
+---@param parameters tts__SnapPoint[]
+---@return string
+function setSnapPoints(parameters) end
```

Comment:
Looks like it was already this way. However, could we rename `parameters` to `snapPoints`?